src/UI/Components/Diagram/shapes.ts

```diff
@@ -297,13 +346,28 @@ export class ServiceEntityBlock extends shapes.standard.HeaderedRecord {
     return this;
   }
 
-  editColumns(data: Array<ColumnData>, shouldBeCollapsed = true) {
+  /**
+   * Edits the columns of the entity and optionally sets the collapsed state.
+   *
+   * @param {Array<ColumnData>} data - The array of column data to set.
+   * @param {boolean} [shouldBeCollapsed=true] - Flag indicating whether the entity should be collapsed.
+   * @returns {this} The updated entity block.
+   */
+  editColumns(data: Array<ColumnData>, shouldBeCollapsed = true): this {
```

Comment:
Wouln't update be the better term? You're using it in the return description too. Edit is for me a bit different than updating.

Reply:
Just in case as I don't se update here in the code viewer, yet everything is up to date:
![Screenshot 2024-11-06 at 13 21 20](https://github.com/user-attachments/assets/82223bed-3f9b-4bb5-b3fc-fad37562d26f)